docs/index.md

```diff
@@ -0,0 +1,146 @@
+# VSUP: Value-Suppressing Uncertainty Palettes
+
+A Python package for visualizing data with uncertainty using Value-Suppressing Uncertainty Palettes (VSUPs). Inspired by [https://github.com/uwdata/vsup](https://github.com/uwdata/vsup).
+
+## Installation
+
+```bash
+pip install vsup
+```
+
+### Development
+
+The project is developed with [uv](https://docs.astral.sh/uv/).
+
+To check for a local python environment, run:
+
+```bash
+uv run python
+```
+
+Also install the [pre-commit](https://pre-commit.com/) hooks with:
+
+```bash
+uv tool install pre-commit
+pre-commit install
+```
+
+## Usage
+```python
+from vsup import VSUP
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Create a grid of values and uncertainties for better visualization
+n_points = 50
+step = 1 / n_points
+values = np.linspace(step / 2, 1 - step / 2, n_points)
+uncertainties = np.linspace(step / 2, 1 - step / 2, n_points)
+
+# Create a 2D grid
+values, uncertainties = np.meshgrid(values, uncertainties)
+
+# Colorize the data
+axs = plt.subplots(3, 3, figsize=(9, 9))[1]
+
+for row, quantization in zip(axs, [None, "linear", "tree"]):
+    for ax, mode in zip(row, ["us", "ul", "usl"]):
+        vsup = VSUP(palette="flare", mode=mode, quantization=quantization)
+
+        colors = vsup(values, uncertainties)
+        ax.pcolormesh(values, uncertainties, colors)
+        # ax.set_title(f"{mode}")  #\n({description})")
+        ax.set_xlabel("Value")
+        ax.set_ylabel("Uncertainty")
+```
+
+## Example
+
+```python {marimo}
+import marimo as mo
+
+import micropip
+await micropip.install(["vsup", "matplotlib", "numpy"])
+
+from matplotlib import colormaps
+
+seaborn_colors = ["flare"]
+matplotlib_colors = list(colormaps)
+colors = seaborn_colors + matplotlib_colors
```

Comment:
The list can be restricted here. Definitely some of the them looks not so good 😆